### PR TITLE
Update primes example and re-run.

### PR DESCRIPTION
**Pull request by MarkDaoust**

### CODE REVIEW

File: quickstarts/Code_Execution.ipynb [hunk 1]
```diff
@@ -57,7 +57,7 @@
     },
```
Comment (markmcd) [inline quickstarts/Code_Execution.ipynb:57]: <p>Clear error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (markmcd) [inline quickstarts/Code_Execution.ipynb:57]: <p>Should these tokens be included? <code>&lt;/del_n7bd&gt;&nbsp;# &lt;del_n7bd&gt;</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

**Review by markmcd — DISMISSED** [review]
I would have just merged this but there are some funky tokens so I thought I'd see what you think?

File: quickstarts/Code_Execution.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply quickstarts/Code_Execution.ipynb:57]: <p>Done</p>

**Pull request merged**